### PR TITLE
NuGetSources task

### PR DESCRIPTION
**Pull request by darsen**
Added task for nuget sources command.

### CODE REVIEW

File: src/main/groovy/com/ullink/NuGetPlugin.groovy [hunk 1]
```diff
@@ -27,6 +27,11 @@ class NuGetPlugin implements Plugin<Project> {
             group = BasePlugin.UPLOAD_GROUP
             description = 'Pushes the NuGet package to the configured server url.'
         }
+
+        project.task('nugetSources', type: NuGetSources) {
+            group = BasePlugin.UPLOAD_GROUP
```
Comment (gluck) [inline src/main/groovy/com/ullink/NuGetPlugin.groovy]: probably this operation shouldn't be part of the upload group, you can probably remove the group entirely if there's no fitting/existing one

Comment (darsen) [reply src/main/groovy/com/ullink/NuGetPlugin.groovy]: Ok, I have removed the group.

File: src/main/groovy/com/ullink/NuGetSources.groovy [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        }
+    }
+
+    @Override
+    void exec() {
+        args operation
+        if (sourceName) args '-Name', sourceName
+        if (sourceUrl) args '-Source', sourceUrl        
+        if (username) args '-UserName', name
+        if (password) args '-Passsword', name
```
Comment (gluck) [inline src/main/groovy/com/ullink/NuGetSources.groovy]: typo ?

Comment (darsen) [reply src/main/groovy/com/ullink/NuGetSources.groovy]: Yes, sorry about that. Should be OK now.

File: src/main/groovy/com/ullink/NuGetSources.groovy [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            }
+        }
+    }
+
+    @Override
+    void exec() {
+        args operation
+        if (sourceName) args '-Name', sourceName
+        if (sourceUrl) args '-Source', sourceUrl        
+        if (username) args '-UserName', name
```
Comment (gluck) [inline src/main/groovy/com/ullink/NuGetSources.groovy]: should be username ?

File: src/main/groovy/com/ullink/NuGetSources.groovy [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        if (password) args '-Passsword', password
```
Comment (gluck) [inline src/main/groovy/com/ullink/NuGetSources.groovy]: typo still (3 's')

File: src/main/groovy/com/ullink/NuGetSources.groovy [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        if (configFile) args '-ConfigFile', configFile
+        if (storePaswordInClearText) args '-StorePaswordInClearText'
```
Comment (gluck) [inline src/main/groovy/com/ullink/NuGetSources.groovy]: typo as well (`pas` vs `pass`)

**Comment by darsen** [comment]
Sorry about that. Typos fixed.

File: src/main/groovy/com/ullink/NuGetSources.groovy [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        }
+    }
+
+    @Override
+    void exec() {
+        args operation
+        if (sourceName) args '-Name', sourceName
+        if (sourceUrl) args '-Source', sourceUrl        
+        if (username) args '-UserName', username
+        if (password) args '-Passsword', password
```
Comment (gluck) [inline src/main/groovy/com/ullink/NuGetSources.groovy]: Still the '-Passs' typo here, no ?

**Comment by darsen** [comment]
Yes, I have fixed that one now, thank you.

**Pull request merged**